packages/app/src/App.tsx

```diff
@@ -52,8 +52,35 @@ import { ToolboxPage } from '@drodil/backstage-plugin-toolbox';
 import { ExplorePage } from './components/explore/ExplorePage';
 import { NewRelicPage } from '@backstage/plugin-newrelic';
 
+import { githubAuthApiRef } from '@backstage/core-plugin-api';
+import { googleAuthApiRef } from '@backstage/core-plugin-api';
```

Comment:
can be merged with the line above

Reply:
did you try `yarn lint` ?

Reply:
yep but nothing flagged:
![image](https://github.com/Zenika/backstage-demo/assets/34473421/aa70f92b-8cf1-4e9c-a8a9-b4c9f31f0533)

Reply:
:/ and `yarn tsc`and `yarn build`?